checker/checkers.go

```diff
@@ -110,6 +108,49 @@ func AssetCheckerCreators(assetPaths ...string) ([]Creator, []okgo.ConfigUpgrade
 	return checkerCreators, configUpgraders, nil
 }
 
+type typeAndPriority struct {
+	checkerType     okgo.CheckerType
+	checkerPriority okgo.CheckerPriority
+}
+
+func determineTypeAndPriorityForPaths(assetPaths []string) (map[string]typeAndPriority, error) {
+	typeAndPriorities := map[string]typeAndPriority{}
```

Comment:
nit: prefer `make` for initializing map to be populated

```suggestion
	typeAndPriorities := make(map[string]typeAndPriority)
```

Reply:
Done